platform/debian/do_debian_package.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+
+# Script to generate debian packages for KOReader.
+PACKAGE="KOReader"
+AUTHOR="KOReader community"
+DESC_1="An ebook reader application supporting PDF, DjVu, EPUB, FB2 and many more formats"
+DESC_2="https://github.com/koreader/koreader"
```

Comment:
Probably better to use:
```suggestion
DESC_2="https://koreader.rocks"
```

Reply:
updated homepage :+1: